marimo/_server/server.py

```diff
@@ -318,9 +322,7 @@ async def start_server(
     _loggers.initialize_tornado_loggers(development_mode)
     signal.signal(signal.SIGINT, interrupt_handler)
 
-    root = os.path.realpath(
-        importlib_resources.files("marimo").joinpath("_static")
-    )
+    root = os.path.realpath(importlib_files("marimo").joinpath("_static"))
```

Comment:
To appease the type-checking error on Python 3.10:

```suggestion
    root = os.path.realpath(str(importlib_files("marimo").joinpath("_static")))
```

Reply:
I missed this too. Will run mypy as well in the future :) Would you like to set up a pre-commit hook or include it in the Makefile?

Reply:
No worries! We have it as a check in the Makefile: `make py-check` formats, lints, and runs mypy. Maybe in the future we can consider a pre-commit. Hard to get 100% coverage locally though, since the typechecker and tests need to run on multiple Python versions.